File: sdk/kanari_pay/lib/src/models/environment.dart
enum KanariEnvironment {
  local('http://127.0.0.1:6767', 'http://127.0.0.1:3000'),
  dev('http://192.168.1.102:19001', 'http://192.168.1.101:3000');

  final String url;
  final String authUrl;

  const KanariEnvironment(this.url, this.authUrl);

  /// Returns the RPC endpoint URL
  String get rpcUrl => '$url/rpc';
}

File: sdk/kanariapp/app/src/main/java/com/jamesatomc/kanariapp/network/models/Environment.kt
package com.jamesatomc.kanariapp.network.models

enum class KanariEnvironment(
    val url: String,
    val authUrl: String
) {
    local("http://10.0.2.2:6767/", "http://10.0.2.2:3000/"), 
    dev("http://192.168.1.102:19001/", "http://10.0.2.2:3000/");

    val rpcUrl: String get() = "${url.removeSuffix("/")}/rpc"
    val baseUrl: String get() = url
}